Readme.md

```diff
@@ -1 +1,57 @@
-# Discover
+# Liquid Investigations: Node Discovery Daemon
+
+This repository implements a method finding out in real time what other Liquid Investigations nodes are online.
```

Comment:
Maybe link to the organization's page, `https://github.com/liquidinvestigations`?